GameTests.java

```diff
@@ -66,19 +66,20 @@ public void gameEndsAfterWin() {
     @Test
     public void gameEndsAfterDraw() {
         InputStream stream = new ByteArrayInputStream("9".getBytes());
-        Board board = new Board("xxooxxox-");
+        Board board = new Board("xxoxoxxo-");
         Game game = new Game(stream, board);
         game.assignPlayers();
-        assertThat(game.playerMakesMove().getState(), is("Game Over"));
+        assertThat(game.playerMakesMove().getState(), is("Draw"));
```

Comment:
This assertion isn't testing what the test proposes: 'gameEndsAfterDraw', it is testing gameIsADraw.
What you are testing is state not behaviour, which I think is okay in this instance, but can you think how to test behaviour of the game ending after a draw?